server/arweave/router.ts

```diff
@@ -70,7 +70,7 @@ router.post('/upload',
       res.json({
         arweaveId: existingArweaveId,
         ipfsHash,
-        numAssetsIds,
+        numAssetsIds: numAssetsIds || [],
```

Comment:
I wonder in what case `await registerNUMAssets` would return empty array though...

Reply:
Oops, my bad 😓